sanity/schemas/components/formCta.js

```diff
@@ -0,0 +1,40 @@
+export default {
+  name: "formCta",
+  title: "Form call to action ",
+  type: "object",
+  fields: [
+    {
+      name: 'theme',
+      type: 'string',
+      title: 'Typ',
+      options: {
+        list: [
+          { title: 'Primary', value: 'primary' },
+          { title: 'Secondary', value: 'secondary' }
+        ],
+        layout: 'radio',
+        direction: "horizontal"
+      },
+      initialValue: 'primary',
+    },
+    {
+      type: 'string',
+      name: 'text',
+      title: 'Text',
+      description: 'The text that will appear on the button',
+    }
+  ],
+  preview: {
+    select: {
+      title: 'text',
+      theme: 'theme',
+      href: 'href'
```

Comment:
Here you pick up a `href` attribute, but doesn't use it later in prepare. Either use it below in `prepare` or remove it.